### PR TITLE
refactor: remove es6 methods

### PR DESCRIPTION
**Pull request by dongwoo-kim**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/helper/common.ts [hunk 1]
```diff
@@ -49,17 +49,50 @@ export function findOffsetIndex(offsets: number[], targetOffset: number) {
...
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function pipe(initVal: any, ...args: Function[]) {
+export function pipe<T>(initVal: T, ...args: Function[]) {
   return args.reduce((acc, fn) => fn(acc), initVal);
 }
 
+// eslint-disable-next-line consistent-return
+export function find<T>(pred: (item: T) => boolean, arr: T[]) {
```
Comment (seonim-ryu) [inline src/helper/common.ts]: `pred`는 `predicate`의 약자인가요..?
그냥 `predicate` 쓰는건 어떨까요??

File: src/helper/dom.ts [hunk 2]
```diff
@@ -120,7 +120,7 @@ export function cls(...names: (ClassNameType | [boolean, ClassNameType])[]) {
 }
 
 export function hasClass(el: HTMLElement, className: ClassNameType) {
-  return el.className.split(' ').includes(cls(className));
+  return el.className.split(' ').indexOf(cls(className)) >= 0;
```
Comment (seonim-ryu) [inline src/helper/dom.ts]: `indexOf`를 사용하면 `> -1`로 비교를 하고싶은건 기분탓이겠죠ㅎㅎ

**Comment by seonim-ryu** [comment]
[5/15] 고생하셨습니다. 잘 쓰겠습니다ㅎ (`omit`은 없나요?)

File: src/helper/dom.ts (hunk 2 above)
Comment (jung-han) [reply src/helper/dom.ts]: `!== -1`은 어떤가요ㅎㅎㅎ

File: src/helper/common.ts [hunk 3]
```diff
@@ -49,17 +49,50 @@ export function findOffsetIndex(offsets: number[], targetOffset: number) {
...
+export function some<T>(pred: (item: T) => boolean, arr: T[]) {
+  return !!find(pred, arr);
+}
+
+export function someProp<T>(propName: keyof T, value: T[keyof T], arr: T[]) {
+  return !!findProp(propName, value, arr);
+}
+
+export function findIndex<T>(pred: (item: T) => boolean, arr: T[]) {
+  for (let i = 0, len = arr.length; i <= len; i += 1) {
```
Comment (jung-han) [inline src/helper/common.ts]: 배열의 마지막 인덱스 이상을 접근하는 것 같습니다. `i < len` 이 되어야 할 것 같습니다.

File: src/helper/dom.ts (hunk 2 above)
Comment (js87zz) [reply src/helper/dom.ts]: 저도 `!== -1`이 좋아요ㅋㅋㅋ취향차이...

**Comment by js87zz** [comment]
리뷰완료입니다!고생하셨습니다~

File: src/helper/dom.ts (hunk 2 above)
Comment (dongwoo-kim) [reply src/helper/dom.ts]: `!== -1` 가 낫겠네요 ㅎ

File: src/helper/common.ts (hunk 3 above)
Comment (dongwoo-kim) [reply src/helper/common.ts]: 어머.. ㅋㅋ 네

File: src/helper/common.ts (hunk 1 above)
Comment (dongwoo-kim) [reply src/helper/common.ts]: 음.. 그럴게요 ㅎㅎ

**Pull request merged**